package.json

```diff
@@ -5,9 +5,9 @@
   "repository": "git://github.com/airtable/airtable.js.git",
   "private": false,
   "dependencies": {
-    "async": "0.9.0",
-    "request": "2.42.0",
-    "lodash": "2.4.1"
+    "async": "^1.5.2",
```

Comment:
Thanks for your submission!

Could you change `async` and `request` to specify exact versions (`1.5.2` and `2.73.0` respectively). And let's leave lodash at 2.4.1 since the jump to v4 introduces breaking changes.